src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,47 @@
+import {removeTestUser} from '@webex/test-users';
+
+import {createIntegrationTestUser, createIntegrationSdkInstance} from './testHelper';
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter Interface', () => {
+  let roomsSDKAdapter;
+  let sdk;
+  let user;
+  let createdRoom;
+
+  beforeAll(async () => {
+    // Setting timeout higher for user creation and room creation
+    jest.setTimeout(15000);
```

Comment:
Since there is already a sleep in the user creation, is this also necessary? Why would creating an SDK instance would increase the idling?

Reply:
Ok, now I understand this is NOT the same thing as Javascript's `setTimeout` but I'm still confused as to what it does.
Does it set an _error_ timeout for test? e.g. "if the test doesn't terminate in X sec move on"
or does it set the interval to wait between tests? e.g. "test 1 executed ...wait 1.5 secs ... execute test 2, etc..."

Reply:
Jest will simply fail if an async operation doesn't complete in the 5 seconds that is defaulted. The purpose of this change is to bump up that wait time so the test user creation has time to complete.

Instead of the confusion that `jest.setTimeout` is causing, perhaps we should utilize the `timeout` second param for `beforeAll` as described here: https://jestjs.io/docs/en/api.html#beforeallfn-timeout

Thoughts @lalli-flores ?

Reply:
Since the room requests may take more than 5 seconds, I've abandoned the timeout param idea and moved the timeout to the top of the main describe.